assets/src/edit-story/app/snackbar/snackbar.js

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const Container = styled.div`
+  display: flex;
+`;
+
+const Main = styled.div`
+  background-color: ${({ theme }) => theme.colors.bg.v0};
+  border-radius: 4px;
+  color: ${({ theme }) => theme.colors.fg.v1};
+  max-width: 456px;
+  display: flex;
+  flex-grow: 1;
+  justify-content: center;
+  align-items: baseline;
+  padding: 0;
+  margin-bottom: ${({ position }) =>
+    position.indexOf('bottom') === 0 ? '0.5em' : 0};
+  margin-top: ${({ position }) =>
+    position.indexOf('top') === 0 ? '0.5em' : 0};
+  font-size: ${({ theme }) => theme.fonts.body2.size};
+  line-height: ${({ theme }) => theme.fonts.body2.lineHeight};
+  flex-direction: row;
+`;
+
+const Content = styled.div`
+  padding: 16px;
+  flex-grow: 1;
+`;
+
+const BoldSpan = styled.span`
```

Comment:
Seems a bit pointless. Why not use good old HTML, `<b></b>`?

Reply:
Or a `<strong>` tag?

Reply:
used `@wordpress/element` package and some webpack config from #707